### PR TITLE
platform dashboard

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*

- bcgov/entity/issues/

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-225-bpj0znwh.web.app

File: strr-platform-web/app/pages/platform/application.vue [hunk 1]
```diff
@@ -25,14 +25,16 @@ const platFeeSm = ref<ConnectFeeItem | undefined>(undefined)
 const platFeeLg = ref<ConnectFeeItem | undefined>(undefined)
 const platFeeWv = ref<ConnectFeeItem | undefined>(undefined)
 onMounted(async () => {
-  const smallFee = getFee(ConnectFeeEntityType.STRR, ConnectFeeCode.STR_PLAT_SM)
-  const largeFee = getFee(ConnectFeeEntityType.STRR, ConnectFeeCode.STR_PLAT_LG)
-  const waivedFee = getFee(ConnectFeeEntityType.STRR, ConnectFeeCode.STR_PLAT_WV)
-  platFeeSm.value = await smallFee
-  platFeeLg.value = await largeFee
-  platFeeWv.value = await waivedFee
+  const [smallFeeResp, largeFeeResp, waivedFeeResp] = await Promise.all([
```
Comment (deetz99) [inline strr-platform-web/app/pages/platform/application.vue:28]: nice

File: strr-platform-web/app/stores/platform.ts [hunk 2]
```diff
@@ -4,17 +4,30 @@ export const useStrrPlatformStore = defineStore('strr/platform', () => {
...
     // TODO: add error handling / modal popup?
-    const resp = await $strrApi('/registrations', { method: 'GET' }) as { registrations: PlatformRegistrationPayload[] }
+    const resp = await $strrApi('/registrations', { method: 'GET' }) as { registrations: PlatformRegistrationResp[] }
     return resp?.registrations
   }
 
   const getAccountApplications = async () => {
     // TODO: add error handling / modal popup?
-    const resp = await $strrApi('/applications', { method: 'GET' }) as { applications: PlatformApplicationPayload[] }
+    const resp = await $strrApi('/applications', { method: 'GET' }) as { applications: PlatformApplicationResp[] }
```
Comment (deetz99) [inline strr-platform-web/app/stores/platform.ts:30]: should be able to do this the same as $fetch -> $strrApi<{ applications: PlatformApplicationResp[] }>('/applications..

GET is also on by default

**Comment by deetz99** [comment]
Looks pretty good, I would just double check the typing and default method on the $strrApi calls

**Review by deetz99 — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-225-bpj0znwh.web.app

**Pull request merged**